testutils/api/tenantadm.py

```diff
@@ -14,9 +14,11 @@
 
```

Comment:
Once you are at it, please update copyright year here also.

Reply:
done